db.go

```diff
@@ -44,13 +44,113 @@ func init() {
 	db.Exec(`
 		CREATE TABLE IF NOT EXISTS client (
```

Comment:
@cglotr Since the table already exists, this will never run. Should we handle this or should we for now just handle it manually?

Reply:
We should handle it manually, for now, I think since we are still in the pre-release stage. Post-release, we should do it via migration/ALTER TABLE